quarkus-bon-jova-rockstar/pom.xml

```diff
@@ -42,7 +43,7 @@
         <maven.compiler.release>21</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>3.6.4</quarkus.version>
+        <quarkus.version>3.9.1</quarkus.version>
```

Comment:
This can be 3.9.5

Reply:
Actually, 3.10.0 is out

Reply:
Quarkiverse extensions should, in general, build against the lowest Quarkus level that they're compatible with. (We don't document that very well, doh!). Then the ecosystem CI checks compatibility with the latest Quarkus. So since there was a breaking change in the dev UI in 3.9.1 (I think?), it wants to be 3.9.1, and not lower, but also not higher.